api/src/test/java/uk/gov/dhsc/htbhf/claimant/reporting/payload/ReportPropertiesFactoryTest.java

```diff
@@ -93,7 +85,7 @@ void shouldCreateReportPropertiesForNewClaim() {
                 entry("cd7", postcodeData.getOutcode()),
                 entry("cd8", postcodeData.getParliamentaryConstituency()),
                 entry("cd9", postcodeData.getCcg()), // Clinical Commissioning Group
-                entry("cd10", postcodeData.getCodes().getCcg()), // Clinical Commissioning Group code
+                entry("cd10", postcodeData.getCodes().getCcg()), // Clinical Commissioning Group code`
```

Comment:
Slightly spurious addition of ` to the comment :)

Reply:
Not sure how that happened, removed now.